client/core/trade.go

```diff
@@ -842,38 +842,36 @@ func (t *trackedTrade) processCancelMatch(msgMatch *msgjson.Match) error {
 //
 // This method accesses match fields and MUST be called with the trackedTrade
 // mutex lock held for reads.
-func (t *trackedTrade) counterPartyConfirms(ctx context.Context, match *matchTracker) (have, needed uint32, changed, spent, expired bool) {
+func (t *trackedTrade) counterPartyConfirms(ctx context.Context, match *matchTracker) (have, needed uint32, changed, spent, expired bool, err error) {
+	fail := func(err error) (uint32, uint32, bool, bool, bool, error) {
+		return 0, 0, false, false, false, err
+	}
+
 	// Counter-party's swap is the "to" asset.
 	needed = t.wallets.toAsset.SwapConf
 
 	// Check the confirmations on the counter-party's swap. If counterSwap is
 	// not set, we shouldn't be here, but catch this just in case.
 	if match.counterSwap == nil {
-		t.dc.log.Warnf("counterPartyConfirms: No AuditInfo available to check!")
-		return
+		return fail(errors.New("counterPartyConfirms: No AuditInfo available to check!"))
 	}
 
 	wallet := t.wallets.toWallet
 	coin := match.counterSwap.Coin
 
 	_, lockTime, err := wallet.LocktimeExpired(match.MetaData.Proof.CounterContract)
 	if err != nil {
-		t.dc.log.Errorf("Error checking if locktime has expired on taker's contract on order %s, "+
-			"match %s: %v", t.ID(), match, err)
+		return fail(fmt.Errorf("error checking if locktime has expired on taker's contract on order %s, "+
+			"match %s: %w", t.ID(), match, err))
 	}
 	expired = time.Until(lockTime) < 0 // not necessarily refundable, but can be at any moment
 
 	have, spent, err = wallet.SwapConfirmations(ctx, coin.ID(),
 		match.MetaData.Proof.CounterContract, match.MetaData.Stamp)
 	if err != nil {
-		if !errors.Is(err, asset.ErrSwapNotInitiated) {
-			// No need to log an error if swap not initiated as this
-			// is expected for newly made swaps involving contracts.
-			t.dc.log.Errorf("Failed to get confirmations of the counter-party's swap %s (%s) "+
```

Comment:
The static text part of this error is now mostly redundant since caller now has `t.dc.log.Errorf("isSwappable: unable to get counter party confirms: %v", err)` (or a variation with `isRedeemable`).

Reply:
https://github.com/decred/dcrdex/compare/07c173256dd1cf8d228a3e1afa33e6274db94ea6..877d5ef06517fba3597850989e15bf3ae935e215

Removed some text from the caller's log...